### ISSUE TITLE
Icon Request: fa-fidget-spinner

### ISSUE THREAD

**Issue by tomwarner13** [FortAwesome/Font-Awesome#11209]
It's 2017. Y'all are slackin'. Get on it already.

**Comment by tkaetzel** [comment]
please escalate this issue

**Comment by markesejd** [comment]
Yes, I have been looking for a fidget spinner icon forever. I love using font-awesome I need this to happen for my life to be complete

**Comment by 0fficerNasty** [comment]
We would be interested in replacing all current spinner gifs with fidget spinner gifs.

**Comment by garak** [comment]
But it needs to be fa-fidget. Then, you can add fa-spinner if you want

**Comment by Argus9** [comment]
My web projects have been missing this certain something lately. This is it.

**Comment by tomwarner13** [comment]
It's 2018 and this is frankly shameful. Don't know if I'll even bother putting in an icon request for the Fortnite logo.

**Comment by tkaetzel** [comment]
Yes. Unacceptable that this has been swept under the rug. Truly a shame.

At this pace, Unicode will likely have a fidget spinner icon first, so I will begin migrating all my projects’ icons as such.

**Comment by Argus9** [comment]
The fad may be dead but the icon needs to live forever!

**Comment by AlexanderJohnston** [comment]
This is the most important item missing from my toolchain.

**Comment by Wingo5315** [comment]
+1 This would be a good addition!

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
